File: check_keys.py
import re

with open(r'c:\ArchLucid\ArchLucid\ArchLucid.Core\Configuration\ConfigurationKeyCatalog.cs', 'r', encoding='utf-8') as f:
    catalog_code = f.read()

with open(r'c:\ArchLucid\ArchLucid\docs\library\CONFIGURATION_REFERENCE.md', 'r', encoding='utf-8') as f:
    doc_text = f.read()

matches = re.findall(r'E\([^,]*,\s*"([^"]+)"', catalog_code)
for key in matches:
    if f"`{key}`" not in doc_text:
        print(f"Missing in doc: {key}")

File: check_keys.cs
using System;
using System.IO;
using System.Linq;
using System.Text.RegularExpressions;

class Program
{
    static void Main()
    {
        var catalogCode = File.ReadAllText(@"c:\ArchLucid\ArchLucid\ArchLucid.Core\Configuration\ConfigurationKeyCatalog.cs");
        var docText = File.ReadAllText(@"c:\ArchLucid\ArchLucid\docs\library\CONFIGURATION_REFERENCE.md");

        var matches = Regex.Matches(catalogCode, @"E\([^,]*, ""([^""]+)""");
        foreach (Match match in matches)
        {
            var key = match.Groups[1].Value;
            if (!docText.Contains("`" + key + "`"))
            {
                Console.WriteLine("Missing in doc: " + key);
            }
        }
    }
}